### PR TITLE
Improve and modernize LocalRest documentation

### PR DESCRIPTION
**Pull request by Geod24**
We should probably also rename at some point.

CC @denizzzka

### CODE REVIEW

File: source/geod24/LocalRest.d [hunk 1]
```diff
@@ -1,30 +1,68 @@
...
+    accessible under the `ctrl` namespace in the instance.
+    The control interface allows to make the node unresponsive to one or all
+    methods, for some defined time or until unblocked.
+    See `sleep`, `filter`, and `clearFilter` for more details.
+
+    Event_Loop:
+    Server process usually needs to perform some action in an asynchronous way.
+    Additionally, some actions needs to be completed at a semi-regular interval,
+    for example based on a timer.
+    For those use cases, a node should call `runTask` or `sleep`, respectively.
```
Comment (AndrejMitrovic) [inline source/geod24/LocalRest.d:47]: I would also make a note not to call vibe.d's runTask / wait by accident, or just to be sure not to mix localrest & vibe.d code.

Comment (Geod24) [reply source/geod24/LocalRest.d:47]: Added clarification

**Review by AndrejMitrovic — APPROVED** [review]

**Comment by AndrejMitrovic** [comment]
@denizzzka is there anything else that is unclear in the documentation?

**Comment by denizzzka** [comment]
I will say if I notice anything

**Comment by Geod24** [comment]
Ping

**Pull request merged**